sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -224,4 +238,38 @@ protected void dispose(AutoCloseable... closeables) {
             }
         }
     }
+
+    static Stream<Arguments> messagingEntityProvider() {
```

Comment:
I am moving common ,between sync and async client,   part in IntegrationTestBase